### PR TITLE
docs(core): fix content errors

### PR DESCRIPTION
**Pull request by petebacondarwin**
This fixes all the "dgeni" errors for the `@angular/core` API docs.

### CODE REVIEW

**Comment by petebacondarwin** [comment]
@IgorMinar I think this should be cherry picked to stable. Is that correct?

File: packages/core/src/metadata/ng_module.ts [hunk 1]
```diff
@@ -105,7 +103,7 @@ export interface NgModule {
   /**
    * Specifies a list of directives/pipes that belong to this module.
    *
-   * ### Example
+   * **Example**:
```
Comment (gkalpak) [inline packages/core/src/metadata/ng_module.ts]: The `:` seems inconsistent with other similar changes (in other PRs :grin:)

File: packages/core/src/metadata/directives.ts [hunk 2]
```diff
@@ -205,7 +203,7 @@ export interface Directive {
   /**
    * Specify the events, actions, properties and attributes related to the host element.
    *
-   * ## Host Listeners
+   * **Host Listeners:**
```
Comment (gkalpak) [inline packages/core/src/metadata/directives.ts]: `:`?

File: packages/core/src/metadata/directives.ts [hunk 3]
```diff
@@ -458,24 +450,21 @@ export interface ComponentDecorator {
    * * **templateUrl** - url to an external file containing a template for the view
    * * **viewProviders** - list of providers available to this component and its view children
    *
-   * ### Example
+   * **Example:**
```
Comment (gkalpak) [inline packages/core/src/metadata/directives.ts]: You have to decide: Which side of `**` does the `:` go? 😛

File: packages/core/src/application_ref.ts [hunk 4]
```diff
@@ -196,8 +196,7 @@ export class PlatformRef {
    * Creates an instance of an `@NgModule` for the given platform
    * for offline compilation.
    *
-   * ## Simple Example
-   *
+   * ** The following example demonstrates calling `bootstrapModuleFactory`:**
```
[the remark this answers is not in the document]
Comment (petebacondarwin) [reply packages/core/src/application_ref.ts]: 😱

File: packages/core/src/metadata/directives.ts (hunk 3 above)
Comment (petebacondarwin) [reply packages/core/src/metadata/directives.ts]: Perhaps I should have written a script ??

**Pull request closed without merging**